templates/matching.jsx

```diff
@@ -49,8 +49,8 @@ export default function Matching(props) {
           const activeOption = _options.find(option => (option._itemIndex === _index) && option._isActive);
           const displayItemAsCorrect = (!_isEnabled && _shouldShowMarking && (_isCorrectAnswerShown || activeOption?._shouldBeSelected));
           const questionTitleId = `${_id}-matching-item-${_index}__title`;
-          const questionTextId = `${_id}-matching-item-${_index}__text`;
           const hasMultipleCorrectAnswers = _correctAnswers.length > 1;
+          const ariaLabel = text ? questionTitleId : (ariaQuestion ? null : `${_id}-header`);
```

Comment:
```suggestion
          const ariaLabelledBy = text ? questionTitleId : (ariaQuestion ? null : `${_id}-header`);
```
The variable name `ariaLabel` isn't correct here, it's the `aria-labelledby` id that you're passing through, so perhaps change to make easier to read?